ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/UserManagementServiceIT.scala

```diff
@@ -396,6 +396,7 @@ final class UserManagementServiceIT extends LedgerTestSuite {
     allocate(NoParties),
     enabled = _.userManagement.maxUsersPageSize > 0,
     disabledReason = "requires user management feature with users page size limit",
+    runConcurrently = false,
```

Comment:
This test creates maxUsersPageSize of users and should be run serially so that it can clean those users before anyone can see it